### PR TITLE
Now supports cloud Metadefender

### PR DESCRIPTION
**Pull request by dem-evgeny**

### CODE REVIEW

File: Integrations/integration-opswat-metadefender.yml [hunk 1]
```diff
@@ -39,42 +39,82 @@ configuration:
   defaultvalue: "66"
   type: 0
   required: true
+- display: It is cloud
```
Comment (Itay4) [inline Integrations/integration-opswat-metadefender.yml]: I would change it to something which looks a bit more professional, like `Cloud based`

File: Integrations/integration-opswat-metadefender.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ configuration:
   defaultvalue: ""
   type: 4
   required: false
-- display: Trust any certificate (unsecure)
```
Comment (Itay4) [inline Integrations/integration-opswat-metadefender.yml]: Why did you change it? Should be as it was

File: Integrations/integration-opswat-metadefender.yml [hunk 3]
```diff
@@ -39,42 +39,82 @@ configuration:
   defaultvalue: "66"
   type: 0
   required: true
+- display: It is cloud
+  name: cloud
+  defaultvalue: "false"
```
Comment (Itay4) [inline Integrations/integration-opswat-metadefender.yml]: Instances which are already defined won't get the default value, you should handle it in the code.

File: Integrations/integration-opswat-metadefender.yml [hunk 4]
```diff
@@ -39,42 +39,82 @@ configuration:
...
+  type: 8
+  required: false
 script:
   script: |
     var sendRequest = function(url,param, fileId) {
         var requestUrl = params.url + url + param;
```
Comment (Itay4) [inline Integrations/integration-opswat-metadefender.yml]: Please handle the case that the URL given in the parameters is followed by `/`

File: Integrations/integration-opswat-metadefender.yml [hunk 5]
```diff
@@ -39,42 +39,82 @@ configuration:
...
+                            Accept: ['application/json']
+                        }
                     }
-                }
-                );
+                    );
+            }
         }
 
+        return res;
```
Comment (Itay4) [inline Integrations/integration-opswat-metadefender.yml]: Remove this

File: Integrations/integration-opswat-metadefender.yml [hunk 6]
```diff
@@ -39,42 +39,82 @@ configuration:
...
+                    requestUrl,
+                    {
+                        Method: 'GET',
+                        Headers: {
+                            Accept: ['application/json']
+                        }
                     }
-                }
-                );
+                    );
```
Comment (Itay4) [inline Integrations/integration-opswat-metadefender.yml]: Please lint properly

File: Integrations/integration-opswat-metadefender.yml (hunk 1 above)
Comment (dem-evgeny) [reply Integrations/integration-opswat-metadefender.yml]: Changed

**Comment by dem-evgeny** [comment]
@Itay4 "Did you test it?"
Strange question. Yes, I did. Not only I but SE as well.

File: Integrations/integration-opswat-metadefender.yml (hunk 2 above)
Comment (dem-evgeny) [reply Integrations/integration-opswat-metadefender.yml]: Because it should be: "Do not validate server certificate (insecure)"

File: Integrations/integration-opswat-metadefender.yml (hunk 3 above)
Comment (dem-evgeny) [reply Integrations/integration-opswat-metadefender.yml]: I already handled it:
"if (params.cloud) "
It will go to else

File: Integrations/integration-opswat-metadefender.yml (hunk 5 above)
Comment (dem-evgeny) [reply Integrations/integration-opswat-metadefender.yml]: Removed

File: Integrations/integration-opswat-metadefender.yml (hunk 4 above)
Comment (dem-evgeny) [reply Integrations/integration-opswat-metadefender.yml]: Added

File: Integrations/integration-opswat-metadefender.yml (hunk 2 above)
Comment (Itay4) [reply Integrations/integration-opswat-metadefender.yml]: No, we changed it.

File: Integrations/integration-opswat-metadefender.yml (hunk 3 above)
Comment (Itay4) [reply Integrations/integration-opswat-metadefender.yml]: `params.cloud` will be `undefined` for existing instances, as they won't get the default value you set for it. You should handle that before.

Comment (dem-evgeny) [reply Integrations/integration-opswat-metadefender.yml]: And if it will be undefined to where it will go? To else...

Comment (Itay4) [reply Integrations/integration-opswat-metadefender.yml]: Yes, but it's bad practice.

File: Integrations/integration-opswat-metadefender.yml (hunk 2 above)
Comment (dem-evgeny) [reply Integrations/integration-opswat-metadefender.yml]: Changed

File: Integrations/integration-opswat-metadefender.yml (hunk 6 above)
Comment (dem-evgeny) [reply Integrations/integration-opswat-metadefender.yml]: Chnaged

**Comment by dem-evgeny** [comment]
@Itay4 thanks :)
